staging/src/k8s.io/apiserver/pkg/storage/errors.go

```diff
@@ -171,30 +171,10 @@ func NewInternalErrorf(format string, a ...interface{}) InternalError {
 	return InternalError{fmt.Sprintf(format, a...)}
 }
 
-var tooLargeResourceVersionCauseMsg = "Too large resource version"
-
 // NewTooLargeResourceVersionError returns a timeout error with the given retrySeconds for a request for
 // a minimum resource version that is larger than the largest currently available resource version for a requested resource.
 func NewTooLargeResourceVersionError(minimumResourceVersion, currentRevision uint64, retrySeconds int) error {
 	err := errors.NewTimeoutError(fmt.Sprintf("Too large resource version: %d, current: %d", minimumResourceVersion, currentRevision), retrySeconds)
-	err.ErrStatus.Details.Causes = []metav1.StatusCause{{Message: tooLargeResourceVersionCauseMsg}}
+	err.ErrStatus.Details.Causes = []metav1.StatusCause{{Type: metav1.CauseTypeResourceVersionTooLarge}}
 	return err
 }
-
-// IsTooLargeResourceVersion returns true if the error is a TooLargeResourceVersion error.
-func IsTooLargeResourceVersion(err error) bool {
```

Comment:
if we want to backport this, I would probably leave this method here and delegate to `apierrors.IsTooLargeResourceVersion(err)` in old releases, rather than remove an exported method

Reply:
done